lego/apps/events/migrations/0023_auto_20190319_0007.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.1.7 on 2019-03-19 00:07
```

Comment:
merge these migrations

Reply:
`TBA` should be default for new event, while old events should be migrated ti get `NORMAL`.